community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/state/storeview/LabelScanViewIdIterator.java

```diff
@@ -34,20 +32,13 @@
  */
 class LabelScanViewIdIterator implements PrimitiveLongResourceIterator
```

Comment:
We need it to close the reader when we are done with iteration. Have any simpler ideas on how to do this?
Nice catch on javadoc. Fixing!